tests/api2/test_007_pool.py

```diff
@@ -146,13 +182,15 @@ def test_08_import_msdosfs():
     expect_state(job_id, "SUCCESS")
 
 
-def test_09_look_if_Directory_slash_File():
+def test_11_look_if_Directory_slash_File(request):
```

Comment:
What is this testing exactly?

Reply:
I am not sure I did not add this test.

Reply:
I move it out of the pool test and move it in pool_import and mark it to skip all pool import test on HA.